src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js

```diff
@@ -92,6 +92,8 @@ describe('Test scenario buttons when scenario is not running', () => {
       expect(saveScenarioButton.Button).not.toBeInTheDocument();
       expect(discardChangesButton.Button).not.toBeInTheDocument();
       expect(stopRunButton.Button).not.toBeInTheDocument();
+      expect(getByDataCy('running-state-spinner')).not.toBeInTheDocument();
+      expect(runningStateLabel.Typography).not.toBeInTheDocument();
```

Comment:
Is it really interesting to create a Typography class just for using it as a simple _component.getBydataCy()_ ?
Besides, I do not see any use of _text_

Reply:
I created it for future unit tests that could use a _Typography_ class as it's one our most used components. And _text_ is a remaining of an attempt to test `textContent` but as it's tricky and not so important, I gave up and kept _text_ for future UT :slightly_smiling_face: maybe we need a vote for it, @csm-thu, your opinion ? :smile:

Reply:
No objection to create genreric classes to make UT easier to write, as long as the tested component is used several times. `Typography` seems to be a good candidate for such a generic class :smile: